src/commands/txCommands/pull.js

```diff
@@ -0,0 +1,70 @@
+const chalk = require('chalk');
+const fs = require('fs');
+const Rx = require('rxjs');
+const path = require('path');
+const paths = require('../../util/paths');
+const txlib = require('./util/index');
```

Comment:
The node `require` convention is to just reference the directory name when you're targeting the `index.js` file it contains - this line is equivalent to

```js
const txlib = require('./util');
```

It sort of feels more ambiguous, but I think it's the standard in node land